### PR TITLE
Create test.txt

### PR DESCRIPTION
**Pull request by sindhucj**

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (np14) [inline test.txt:1]: Some comment

**Review by np14 — CHANGES_REQUESTED** [review]
Please change it to .....

**Comment by sindhucj** [comment]
Changes incorporated

File: test.txt (hunk 1 above)
Comment (sindhucj) [reply test.txt:1]: test

**Review by np14 — APPROVED** [review]
Approved

**Pull request merged**